config/values/bursaries.yml

```
@@ -0,0 +1,25 @@
+bursaries:
+  SKE: £175
```

```suggestion
  SKE-weekly: £175
```
worth giving more context?